zlib-rs/src/deflate.rs

```diff
@@ -841,15 +841,18 @@ impl Value {
         self.a
     }
 
-    pub(crate) fn code(self) -> u16 {
+    #[inline(always)]
```

Comment:
Usage of `#[inline(always)]` is discouraged unless you can actually prove that it improves performance, but it isn't all that important either.